cypress/objects/loginPage.js

```diff
@@ -0,0 +1,42 @@
+class LoginPage {
+    visit() {
+        cy.visit('/'); 
+    }
+
+    getUserValidationIndicator() {
+        return cy.get('.Select.not-valid')
+    }
+
+    getSubmitButton() {
+        return cy.get('[type="submit"]')
+    }
+
+    getSpecificUserFromDropDown(userName) {
+        return cy.get(`[aria-label="${userName}"]`)
+    }
+
+    getUserDropDown() {
+        return cy.get('[id="loginForm.userId"]')
+    }
+
+    getRoleDropDown() {
+        return cy.get('[id="loginForm.role"]')
+    }
+
+    getSpecificRoleFromDropDown(role) {
+        return cy.get(`[aria-label="${role}"]`)
+    }
+    
+    getUserTitle() {
```

Comment:
Methods like these could be put in `common.js` or` pageComponents.js `files, because this is shared between pages and if you would want to check username or navigation bar it would look weird if you have to use `loginPage `to do so.

Reply:
This is not yet fixed